modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm

```diff
@@ -9,35 +9,35 @@
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/civil_defense
 	item_type = /obj/item/storage/medkit/civil_defense/stocked
-	cost = PAYCHECK_COMMAND * 2.5
+	cost = PAYCHECK_COMMAND * 5 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/frontier
 	item_type = /obj/item/storage/medkit/frontier/stocked
-	cost = PAYCHECK_COMMAND * 3.5
+	cost = PAYCHECK_COMMAND * 7 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/combat_surgeon
 	item_type = /obj/item/storage/medkit/combat_surgeon/stocked
-	cost = PAYCHECK_COMMAND * 3.5
+	cost = PAYCHECK_COMMAND * 7 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/robo_repair
 	item_type = /obj/item/storage/medkit/robotic_repair/stocked
-	cost = PAYCHECK_COMMAND * 3.5
+	cost = PAYCHECK_COMMAND * 16 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```

Comment:
```suggestion
	cost = PAYCHECK_COMMAND * 7 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```

Reply:
And assuming it's COMMAND * 7

/datum/supply_pack/science/synth_medkits
adjustment to
cost =  CARGO_CRATE_VALUE * 14